lib/tasks/import/wikidata_import_games.rake

```diff
@@ -87,45 +86,26 @@ namespace 'import:wikidata' do
           game_hash[name].uniq!
         end
 
-        pcgamingwiki_id = wikidata_json.dig('P6337')&.first&.dig('mainsnak', 'datavalue', 'value')
-        steam_app_id = wikidata_json.dig('P1733')&.first&.dig('mainsnak', 'datavalue', 'value')
-        epic_games_store_id = wikidata_json.dig('P6278')&.first&.dig('mainsnak', 'datavalue', 'value')
-        # Remove the 'game/' prefix from the GOG.com IDs.
-        gog_id = wikidata_json.dig('P2725')&.first&.dig('mainsnak', 'datavalue', 'value')&.gsub('game/', '')
-        mobygames_id = wikidata_json.dig('P1933')&.first&.dig('mainsnak', 'datavalue', 'value')
-        giantbomb_id = wikidata_json.dig('P5247')&.first&.dig('mainsnak', 'datavalue', 'value')
-
-        release_dates = wikidata_json.dig('P577')&.map { |date| date.dig('mainsnak', 'datavalue', 'value', 'time') }
-        release_dates&.map! do |time|
-          if time.nil?
-            nil
-          else
-            begin
-              Time.parse(time).to_date
-            rescue ArgumentError
-              nil
-            end
-          end
-        end
-        # Set release date equal to nil, or the earliest release date if
-        # all of the release dates above resolved to a proper date. It's done
-        # this way to prevent bad release dates from being used if Wikidata
-        # returns a date like "June 2019", which is represented as "2019-06-00",
-        # an invalid date.
-        release_date = nil
-        release_date = release_dates&.min unless release_dates&.any? { |date| date.nil? }
+        # Grab the earliest release date that we can actually parse
```

Comment:
```suggestion
        # Grab the earliest release date that we can actually parse. It's done
        # this way to prevent bad release dates from being used if Wikidata
        # returns a date like "June 2019", which is represented as "2019-06-00",
        # an invalid date.
```